ibis/backends/tests/test_client.py

```diff
@@ -277,18 +313,36 @@ def test_rename_table(con, temp_table, temp_table_orig):
     ["trino"], reason="trino doesn't support NOT NULL in its in-memory catalog"
 )
 @mark.broken(["snowflake"], reason="snowflake shows not nullable column as nullable")
+@mark.broken(
+    ["flink"],
+    raises=AssertionError,
+    reason=(
+        "assert not True, where True = Int64(nullable=True).nullable"
+        "Flink shows not nullable column as nullable"
+    ),
+)
 def test_nullable_input_output(con, temp_table):
     sch = ibis.schema(
         [("foo", "int64"), ("bar", dt.int64(nullable=False)), ("baz", "boolean")]
     )
-    t = con.create_table(temp_table, schema=sch)
+
+    if con.name == "flink":
+        t = con.create_table(
+            temp_table,
+            schema=sch,
+            tbl_properties={
+                "connector": None,
+            },
```

Comment:
Same

Reply:
Referring to the [reply above](https://github.com/ibis-project/ibis/pull/6920#discussion_r1372456721).